src/js/utils/utils.js

```diff
@@ -44,3 +44,17 @@ export function getLastModifiedDate(intl, modfiedDate) {
     var value = { date: formattedModefiedDate };
     return formatMessage({ id: 'utils.utils.lastmodifieddate' }, value);
 }
+
+export function humanFileSize(bytes) {
```

Comment:
I'll create new cards for analytics and I don't want to put everything in AnaltyicsPage. I'll separate them as components and use this function in them as well. That's why I put it in utils because it'll be used in multiple places.